app/src/main/java/mihon/migration/SetupLibraryUpdateMigration.kt

```diff
@@ -0,0 +1,16 @@
+package mihon.migration
```

Comment:
```suggestion
package mihon.core.migration
```

Reply:
Do we want the actual migrations to be in the same package as the Migrator logic?

Reply:
can put them in a subpackage 🤔